src/pyfaf/retrace.py

```diff
@@ -98,62 +100,88 @@ def __init__(self, db_debug_package, db_src_package, bin_pkg_map, db=None):
 # pylint: enable-msg=R0903
 
 
-class RetraceWorker(threading.Thread, object):
+class RetracePool:
     """
-    The worker providing asynchronous unpacking of packages.
+    A class representing a pool of workers that run the retracing job for given tasks.
     """
 
-    def __init__(self, worker_id, inqueue, outqueue):
-        name = "Worker #{0}".format(worker_id)
-        super(RetraceWorker, self).__init__(name=name)
-        self.inqueue = inqueue
-        self.outqueue = outqueue
-        self.stop = False
-        # Instance of 'RootLogger' has no 'getChildLogger' member
+    def __init__(self, db, tasks, problemplugin, workers):
+
+        self.name = "RetraceWorker"
```

Comment:
RetracePool?

Reply:
Ahh, but then the logs would be `faf.blabla.RetracePool: mumbo jumbo`.

Maybe change the format of the logs to show thread names instead?

Reply:
Yeah, that might be better.

Reply:
I've changed it to `RetracePool` for now. 

I have some changes for logging that I would like solve in different PR.